### ISSUE TITLE
getFileInfo() use .prettierignore as default

### ISSUE THREAD

**Issue by Wells-Codes** [prettier/prettier#13518]
When getting file info using the CLI `prettier --file-info test.js`, it uses `.prettierignore` as the default when you do not provide an ignore-path. I am proposing that the API `getFileInfo()` follow the same behavior and default `options.ignorePath` to `.prettierignore`

Related issues: https://github.com/prettier/prettier/issues/10844

**Environments:**

- Prettier Version: 2.7.1
- Running Prettier via: CLI, Node.js API
- Runtime: Node.js v16
- Operating System: macOS
- Prettier plugins (if any): none

**Steps to reproduce:**
- Use API to call `getFileInfo('test.js')` for a file that is listed in `.prettierignore`
- Use CLI  on the same file `prettier --file-info test.js`

![Screen Shot 2022-09-24 at 11 31 52 AM](https://user-images.githubusercontent.com/19676292/192109295-5ab86d2b-901e-48b0-9d49-7cb877617372.png)
Stackblitz: https://stackblitz.com/edit/prettier-file-info?file=index.js&file=.prettierignore

**Expected behavior:**
CLI and API should return same result when given same inputs

**Actual behavior:**
CLI and API return different results, because CLI has a default ignore path and API does not

**Comment by thorn0** [comment]
Reading the discussion in https://github.com/prettier/prettier/pull/4341 (specifically, this comment - https://github.com/prettier/prettier/pull/4341#issuecomment-386039723), I see that the initial intention was to use `".prettierignore"` as the default value for the method too, not only for CLI. It's not clear whether this wasn't done intentionally. Looks more like an oversight. At least I couldn't find any arguments against doing this. Hopefully, @kachkaev can comment on this.

**Comment by kachkaev** [comment]
Yes, it could be an oversight on my end. Thanks for raising this @Wells-Codes!

It might worth to quickly check whether other methods have `ignorePath` set to `.gitignore` by default. Maybe ,y proposal in https://github.com/prettier/prettier/pull/4341#issuecomment-386039723 was not consistent with the rest of the API and we did not set the default value because of that.
